lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
OK, that last comment really clarified things for me actually. I see two good approaches:

1. More explicitly using composition. In the constructor, do `this[impl] = new EventTargetImpl(options)`, then in the methods, do `this[impl].method.apply(this[impl], arguments)`. The impls are actually classes.

2. Or, just admit they're bags of methods not meant to be used as standalone classes, and use object literals. `Impl.interface.prototype.addEventListener.apply(this, args)` isn't actually any better than `implMethods.addEventListener.apply(this, args)`.